### ISSUE TITLE
unauthorize device

### ISSUE THREAD

**Issue by idozwi** [Jessecar96/SteamDesktopAuthenticator#1058]
Is it possible to unauthorize a device using sda? In mobile you can remove one pc and see its details and stuff.

**Comment by dyc3** [comment]
No. IIRC You can access that functionality via the steam client or a web browser.

**Comment by idozwi** [comment]
> No. IIRC You can access that functionality via the steam client or a web browser.

using the client or browser only allows you to unauthorize all of the devices and not a specific one
